certora/confs/base.conf

```
@@ -0,0 +1,25 @@
+{
+    "assert_autofinder_success": true,
+    "storage_extension_annotation": true,
+    "global_timeout": "7200",
+    "loop_iter": "3",
+    "optimistic_loop": true,
+    "packages": [
+        "ethers=node_modules/ethers",
+        "@safe-global/api-kit=node_modules/@safe-global/api-kit",
+        "axios=node_modules/axios",
+        "@types/node=node_modules/@types/node",
+        "dotenv=node_modules/dotenv",
+        "@safe-global/protocol-kit=node_modules/@safe-global/protocol-kit",
+        "@safe-global/types-kit=node_modules/@safe-global/types-kit",
+        "forge-std=lib/forge-std/src",
+        "solady=lib/solady/src",
+        "@openzeppelin/contracts=lib/openzeppelin-contracts/contracts",
+        "@openzeppelin/contracts-upgradeable=lib/openzeppelin-contracts-upgradeable/contracts"
+    ],
+    "smt_timeout": "7200",
+    "solc_optimize": "200",
+    "solc_via_ir": false,
+    "solc": "solc8.28",
```

```suggestion
    "solc": "solc-0.8.28",
```

I believe that usually in the community people use the official versioning.  See also the suggestion in the workflow yaml file.